src/test/java/Homework18.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import java.time.Duration;
+public class Homework18 extends BaseTest {
+    @Test(priority = 2, enabled = true)
+    public void playSong() {
+        navigateToPage();
+        login("prytulan@gmail.com", "28268286nN*");
+        clickSubmit();
+        String homePage = "https://bbb.testpro.io/#!/home";
+        driver.get(homePage);
+        Actions playReactSong = new Actions(driver);
+        WebElement reactorSong = driver.findElement(By.xpath("//*[@id='homeWrapper']/div/div[1]/section[1]/ol/li[2]/article/span[2]/span[1]"));
```

Comment:
If you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

For example, here is the locator (Copy > Copy Xpath):  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**

For example, here is the correct locator: ` //a[@class='songs']`

It is important to mention that the following principle must be held in deciding locators: the locator must be

- unique
- readable 
- maintainable